### ISSUE TITLE
Second Round of Data Cleaning

### ISSUE THREAD

**Issue by rectheworld** [DataKind-DC/audubon-cbc#40]
# CBC volunteer data needs to be cleaned and organized to work with NOAA Data 

- [ ] Percipiation data is appearing as floats, stings and lists of values. These need to converted into values that can be compared to NOAA 
- [ ]  Check Temperature Unit for conversion accuracy.

**Comment by costa-rica** [comment]
I added 12 columns to the cbc_cleaned_usa_merge.csv (am_rain1, am_rain2, etc.,). For example, 'pm_rain4' column has a '1' for each instance of a '4' in the "pm_rain" column.
[usa_merged_WithNewPrecipCols.zip](https://github.com/DataKind-DC/audubon-cbc/files/4261377/usa_merged_WithNewPrecipCols.zip)

I guess the next step is to figure out how to compare to the NOAA results. I've got the cbc pdf but not very sure which fields are NOAA. I see snow min and max which are probably NOAA. I'll keep looking into that if that helps.
